src/containers/Review/ReviewHome.tsx

```diff
@@ -159,16 +182,63 @@ const getReviewerOptions = (assignments: AssignmentDetails[], currentUserId: num
     },
   ]
   assignments.forEach(({ reviewer: { id, firstName, lastName } }) => {
-    if (!id || !firstName || !lastName) return
+    if (!id || !firstName) return
```

Comment:
I think `username` is a more reliable thing to check than `firstName`.

Reply:
No problem. Changing to `username` for now.